### PR TITLE
Add sso setup guide

### PR DESCRIPTION
**Pull request by cindyliuyn**

### CODE REVIEW

File: workspace/sso_setup.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Single Sign-On (SSO) allows your team to log in to ElevenLabs by using your existing identity provider. This means that your team can use the same credentials which they use for other services to log in to ElevenLabs.
+
+## Choosing identity providers
+To access your SSO settings, click on your profile icon located at the bottom left of the dashboard, select Workspace Settings, and then navigate to the SECURITY & SSO tab.
+
+You can choose from a variety of identity providers, including Google, Okta, GitHub, etc.
+
+<img height="150" src="/workspace/images/identity_providers.png" />
+
+If you disable any of the identity providers, your team members will log out if they're using the disabled providers.
```
Comment (lagercat) [inline workspace/sso_setup.mdx:17]: Maybe we can clarify this a bit:
 
If you disable any of the identity providers, your team members will log out if they're using the disabled providers. They will be able to log back into the same account with any of the allowed providers if they are using the same email. The SSO provider will only appear in that list after it has been configured as shown below.

File: workspace/sso_setup.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Verifying your domain
+Next, you need to verify your domain. This step confirms to ElevenLabs that you own the domain you are configuring for SSO. This is a security measure to prevent unauthorized access to your workspace.
```
Comment (lagercat) [inline workspace/sso_setup.mdx:20]: Could we make it just slightly more clear that we're referring to email domain and not any company domain. Some enterprise customers use a bunch of different ones but it should be whatever domain they expect after the `@` in their company emails.

**Review by lagercat — CHANGES_REQUESTED** [review]
Looks great! Just a few tiny nits and we're good to go.

**Review by lagercat — APPROVED** [review]
LGTM!

**Review by am-holt — APPROVED** [review]

**Pull request merged**